sechub-api-java/src/main/java/com/mercedesbenz/sechub/api/internal/model/AbstractProjectMetaData.java

```diff
@@ -0,0 +1,53 @@
+// SPDX-License-Identifier: MIT
+package com.mercedesbenz.sechub.api.internal.model;
+
+/**
+ * AbstractProjectMetaData is a model class for SecHubClient. It uses internally
+ * the generated class
+ * com.mercedesbenz.sechub.api.internal.gen.model.OpenApiProjectMetaData.<br>
+ * <br>
+ * The abstract wrapper class was generated from a developer with
+ * com.mercedesbenz.sechub.api.generator.AbstractModelFileGenerator and is not
+ * intended to be changed manually!
+ */
+public abstract class AbstractProjectMetaData {
+
+    protected com.mercedesbenz.sechub.api.internal.gen.model.OpenApiProjectMetaData delegate;
+
+    protected AbstractProjectMetaData() {
+        this(null);
+    }
+
+    protected AbstractProjectMetaData(com.mercedesbenz.sechub.api.internal.gen.model.OpenApiProjectMetaData delegate) {
+        if (delegate == null) {
+            this.delegate = new com.mercedesbenz.sechub.api.internal.gen.model.OpenApiProjectMetaData();
+            initDelegateWithDefaults();
+        } else {
+            this.delegate = delegate;
+        }
+    }
+
+    protected void initDelegateWithDefaults() {
+        /* child classes can override this */
+    }
+
+    protected String getStar() {
```

Comment:
is odd - but generated because of open api generator output has it as well.
I remove it from public model, but the internal access (new names for the abstract parts) will still have it unless the open api generators do no longer generate this.